sonar-project.properties

```diff
@@ -4,7 +4,7 @@ sonar.sourceEncoding=UTF-8
 # only analyzing actual library source
 sonar.sources=src
 sonar.tests=src
-sonar.test.inclusions=**/*.test.ts,**/*.test.tsx
+sonar.test.inclusions=**/*.test.ts,**/*.test.tsx,src/test-data/*.ts
```

Comment:
This is test data not tests themselves. Its just exported variables of sample data for test to use. Its ignored because SonarQube will identify the repetitive structures as duplicate code.

Reply:
It's not used by the final artifact, therefor we don't want to bundle it. All files only used for tests need to end with `.test.tsx?` so the bundler knows how to handle them. This isn't an Angular project so it doesn't have tree-shaking.

Reply:
I added the directory to the tsconfig exclude so it shouldnt be an issue.